frontend/bonsai_app/blueprints/cluster/templates/ms_tree.html

```diff
@@ -1,1104 +1,864 @@
-          var angle =
-            y_dif !== 0
-              ? (Math.atan(x_dif / y_dif) / Math.PI) * 180
-              : x_dif === 0
-                ? 0
-                : x_dif > 0
-                  ? 90
-                  : -90;
-          if (y_dif < 0) {
-            angle = 180 + angle;
-          } else if (x_dif < 0) {
-            angle = 360 + angle;
-          }
-          ui.helper
-            .select(".angle-text")
-            .text(Math.round(angle) + "\xB0")
-            .css({
-              "font-size": "1.2em",
-              "font-style": "bold",
-            });
-        });
-
-        $("#search-metadata-icon").click(function (e) {
-          var keyword = $("#search-metadata-input").val();
-          if (keyword) {
-            var ids = the_tree.searchMetadata(
-              keyword,
-              $("#node-label-text").val(),
-            );
-            the_tree.highlightNodes(ids);
-          }
-        });
-
-        $("#graph-div").on("dblclick", function (e) {
-          if (the_tree) the_tree.clearSelection();
-        });
-
-        var target = document.getElementById("modal-header");
-        waiting_spinner = new Spinner({
-          color: "black",
-          lines: 12,
-          top: "13%",
-        }).spin(target);
-
-        // $('.panel-default').find('input, textarea, button, select, span, div').css("opacity", 0.3).prop('disabled',true);
-
-        window.addEventListener("beforeunload", function (e) {
-          if (the_tree) {
-            var confirmationMessage =
-              "All the modifications will lost if you have not saved the GrapeTree as a local file.";
-            (e || window.event).returnValue = confirmationMessage;
-            return confirmationMessage;
-          }
-        });
-
-        $("#file-menu-panel")
-          .css("opacity", 1.0)
-          .addClass("open-menu-panel")
-          .show();
-        $("#button-files").parent().css("opacity", 1.0);
-        $("#button-files")
-          .css("opacity", 1.0)
-          .prop("disabled", false)
-          .trigger("mouseenter");
-
-        $(".mst-menu-title").click(function (e) {
-          if ($(this).prop("disabled") === true) {
-            return;
-          }
-          var id = $(this).attr("id");
-          var this_panel = $("#" + id + "-panel");
-          var icon = $("#" + id + "-icon");
-          if (this_panel.hasClass("open-menu-panel")) {
-            this_panel.toggle("50");
-            this_panel.removeClass("open-menu-panel");
-            icon.attr("class", "glyphicon glyphicon-menu-right");
-          } else {
-            this_panel.toggle("50");
-            this_panel.addClass("open-menu-panel");
-            icon.attr("class", "glyphicon glyphicon-menu-down");
-          }
-        });
-      };
-    </script>
-    <script>
-      let data = {{ data| safe }}
-      loadMSTree(data)
-    </script>
-  </body>
-</html>
+<head>
```

Comment:
Indentation?

Reply:
No. I changed to two spaces for javascript as it makes it a bit more readable